website/content/docs/commands/events.mdx

```diff
@@ -58,9 +58,8 @@ flags](/vault/docs/commands) included on all commands.
 - `-filter` `(string: "")` - Filter expression used to select events to be sent
   through the WebSocket.
 
-Refer to the [Filter expressions](/boundary/docs/concepts/filtering) guide
-in the Boundary documentation for a complete list of filtering options and an
-explanation on how we evaluate filter expressions.
+Refer to the [Filter expressions](/vault/docs/concepts/filtering) guide for a complete
+list of filtering options and an explanation on how we evaluate filter expressions.
```

Comment:
```suggestion
list of filtering options and an explanation on how Vault evaluates filter expressions.
```
Style correction: refer to the specific product/service/etc that takes action. Avoid generalizations like "we" unless speaking about things we (as people) do.